skgstat/interfaces/gstools.py

```diff
@@ -1,46 +1,87 @@
-try:
-    import gstools
-    GSTOOLS_AVAILABLE = True
-except ImportError:
-    GSTOOLS_AVAILABLE = False
+"""GSTools Interface."""
 import numpy as np
 
 
-def __check_gstools_available():  # pragma: no cover
-    if not GSTOOLS_AVAILABLE:
-        print('The gstools interface needs gstools installed.')
-        print("Run 'pip install gstools' to install it.")
-        return False
-    return True
+def stable_rescale(describe):
+    """Get GSTools rescale parameter from sk-gstat stable model description."""
+    return np.power(3, 1 / describe["shape"])
 
 
-def gstools_cov_model(variogram, **kwargs):
-    """GSTools Interface
+def matern_rescale(describe):
+    """Get GSTools rescale parameter from sk-gstat matern model description."""
+    if 0.5 < describe["smoothness"] < 10.0:
+        return 4.0
+    return 6.0
 
-    Pass a :class:`skgstat.Variogram` instance.
-    Returns an **already fitted** variogram model
-    inherited from `gstools.CovModel`. All
-    kwargs passed will be passed to
-    `gstools.CovModel`.
 
+MODEL_MAP = dict(
+    spherical=dict(gs_cls="Spherical"),
+    exponential=dict(gs_cls="Exponential", rescale=3.0),
+    gaussian=dict(gs_cls="Gaussian", rescale=2.0),
+    cubic=dict(gs_cls="Cubic"),
+    stable=dict(
+        gs_cls="Stable", arg_map={"alpha": "shape"}, rescale=stable_rescale
+    ),
+    matern=dict(
+        gs_cls="Matern", arg_map={"nu": "smoothness"}, rescale=matern_rescale
+    ),
+)
+
+
+def skgstat_to_gstools(variogram, **kwargs):
+    """
+    Instantiate a corresponding GSTools CovModel.
+
+    By default, this will be an isotropic model.
+
+    Parameters
+    ----------
+    variogram : :any:`Variogram`
+        Scikit-Gstat Variogram instance.
+    **kwargs
+        Keyword arguments forwarded to the instantiated GSTools CovModel.
+        The default parameters 'dim', 'var', 'len_scale', 'nugget',
+        'rescale' and optional shape parameters will be extracted
+        from the given Variogram but they can be overwritten here.
```

Comment:
I am just wondering if this is actually helpful. On the one hand I think it could be counter-intuitive to fit a skg.Variogram that shows i.e. a specific sill on the plots, but then overwrite the 'var' argument, which effectively makes it another Variogram. On the other hand, it will be the user who is passing in the arguments explicitly, so I guess he has good reasons to do so.
@MuellerSeb, what do you think about that?